kuma/users/templates/users/email/welcome/html.ltxt

```diff
@@ -56,6 +56,7 @@
             </li>
         </ul>
         <p>
+          {# Note: If you change this string also change ``WELCOME_EMAIL_STRINGS`` in kuma/users/tasks.py #}
```

Comment:
I think we need to remove all the `{% trans %}` blocks in this template too, like we did in `plain.ltxt`